src/main/resources/google_checks.xml

```diff
@@ -78,11 +78,6 @@
     <module name="NoLineWrap">
       <property name="tokens" value="PACKAGE_DEF, IMPORT, STATIC_IMPORT"/>
     </module>
-    <module name="EmptyBlock">
-      <property name="option" value="TEXT"/>
-      <property name="tokens"
-               value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH"/>
-    </module>
```

Comment:
removed EmptyBlock though I wasn't able to remove EmptyCatchBlock because it is also used by [6.2](https://checkstyle.org/google_style.html#a6.2)